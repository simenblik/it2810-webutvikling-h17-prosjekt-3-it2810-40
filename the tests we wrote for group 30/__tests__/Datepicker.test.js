import React from 'react';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';
import Datepicker from '../src/Components/Datepicker.js';

/* This is getting a warning. Need something to fix error:
console.error node_modules/fbjs/lib/warning.js:33
      Warning: Failed prop type: Invalid prop `children` of type `array` supplied to `MuiThemeProvider`, expected a single ReactElement.
          in MuiThemeProvider
*/

/* Takes a snapshot of the rendered component and see if it
matches the original snapshot.
*/
describe('<Datepicker/>, functions', () => {
	it('should match to previous snapshots', () => {
		const component = shallow(<Datepicker />);
		const tree = toJson(component);
		expect(tree).toMatchSnapshot();
	});
});
