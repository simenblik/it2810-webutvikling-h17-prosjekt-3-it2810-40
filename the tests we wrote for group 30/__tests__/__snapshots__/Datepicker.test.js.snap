// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Datepicker/>, functions should match to previous snapshots 1`] = `
<div>
  <MuiThemeProvider>
    <DatePicker
      autoOk={false}
      container="dialog"
      disableYearSelection={false}
      disabled={false}
      firstDayOfWeek={1}
      hideCalendarDate={false}
      hintText="Dato"
      openToYearSelection={false}
      style={Object {}}
    />
    <TimePicker
      autoOk={false}
      cancelLabel="Cancel"
      defaultTime={null}
      disabled={false}
      format="24hr"
      hintText="Klokkeslett"
      minutesStep={1}
      okLabel="OK"
      pedantic={false}
      style={Object {}}
      value={null}
    />
  </MuiThemeProvider>
</div>
`;
